src/sgame/sg_bot_skilltree.cpp

```diff
@@ -103,30 +127,32 @@ static void G_SetSkillsetBudgetAliens( int val )
 
 void G_InitSkilltreeCvars()
 {
-	static Cvar::Callback<Cvar::Cvar<std::string>> g_disabledSkillset(
-		"g_disabledSkillset",
-		"Disabled skills for bots, example: " QQ("mantis-attack-jump, prefer-armor"),
+	static Cvar::Callback<Cvar::Cvar<std::string>> g_skillset_disabledSkills(
```

Comment:
About the naming convention, there seem to be three ones:

* `g_skillset_budgetHumans` / `g_skillset_budgetAliens` / `g_skillset_baseSkills` / `g_skillset_disabledSkills` (the current convention in this PR)
* `g_bot_skillsetBudgetHumans` / `g_bot_skillsetBudgetAliens` / `g_bot_baseSkills` / `g_bot_disabledSkillset` (the convention used in #2736
* `g_bot_skillset_budgetHumans` / `g_bot_skillset_budgetAliens` / `g_bot_skillset_baseSkills` / `g_bot_skillset_disabledSkills` (what your comment seemed to suggest)

I can update to the latest version if you want. I'm not fond of the middle one because I think it doesn't make it obvious that the cvars are part of the same group / configuring the same concept.

Reply:
To me it's obviously wrong to use something that doesn't begin with `g_bot_`. I don't really care other than that.

My PR has some additional improvements I'd like to keep: it emits a warning on any misspelled skill, and uses proper data structures instead of strings everywhere.

Reply:
I've added the "g_bot_skillset_" prefix for all cvars